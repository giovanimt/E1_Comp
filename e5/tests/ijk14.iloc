addI rsp, 4 => rsp
addI rsp, 4 => rsp
loadI 393 => r0
addI rfp, 4 => r1
store r0 => r1
addI rfp, 4 => r2
load r2 => r3
addI rfp, 0 => r4
store r3 => r4
