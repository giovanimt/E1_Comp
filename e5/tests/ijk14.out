00032636 393
00032640 393
